src/cachemap.h

```diff
@@ -170,7 +162,6 @@ class CacheMap
     inline void SerializationOp(Stream& s, Operation ser_action)
     {
         READWRITE(nMaxSize);
-        READWRITE(nCurrentSize);
```

Comment:
Does this require any version bumps?

Reply:
Not really, de-serialization simply fails here and corresponding maps are cleared on init, so data will be fetched from the network from scratch. governance.dat will be recreated and filed with new data on node shutdown.